src/plugin/lib-selenium/src/java/org/apache/nutch/protocol/selenium/HttpWebClient.java

```diff
@@ -112,11 +126,16 @@ public static WebDriver getDriverForPage(String url, Configuration conf) {
         }
         LOG.debug("Selenium {} WebDriver selected.", driverType);
   
+        driver.manage().timeouts().pageLoadTimeout(pageLoadWait, TimeUnit.SECONDS);
         driver.get(url);
-        new WebDriverWait(driver, pageLoadWait);
       } catch (Exception e) {
-        throw new RuntimeException(e);
-      }
+			if(e instanceof TimeoutException) {
```

Comment:
@karanjeets please always use the Eclipse code format
https://github.com/apache/nutch/blob/master/eclipse-codeformat.xml